paf-mvp-demo-express/src/websites/portal.ts

```diff
@@ -158,12 +154,15 @@ export const portalWebSiteApp = new VHostApp(name, host);
     }
   });
 
+  const dsaBuilder = new ECDSA_NIT_P256Builder();
```

Comment:
NIT using snake case doesn't seem consistent with the rest of the code base.
Consider using `EcdsaNitP256Builder`, for instance.

Reply:
With a chain of acronyms including numbers, the camel case is not really readable, IMHO. That is why I break the rule here. However, I don't hold strongly.